fiasco/ions.py

```diff
@@ -1528,6 +1651,77 @@ def _karzas_cross_section(self, photon_energy: u.erg, ionization_energy: u.erg,
         cross_section[np.where(photon_energy < ionization_energy)] = 0.*cross_section.unit
         return cross_section
 
+    @u.quantity_input
+    def _gaunt_factor_free_bound_total(self, ground_state=True) -> u.dimensionless_unscaled:
+        r"""
+        The total Gaunt factor for free-bound emission, using the expressions from :cite:t:`mewe_freebound_1986`.
+
+        The Gaunt factor is not calculated for individual levels, except that the ground state has
+        been specified to be :math: `g_{fb}(n_{0}) = 0.9` following :cite:t:`mewe_freebound_1986`.
+
+        Parameters
+        ----------
+        ground_state : `bool`, optional
+            If True (default), calculate the Gaunt factor for recombination onto the ground state :math: `n = 0`.
+            Otherwise, calculate for recombination onto higher levels with :math: `n > 1`.  See Equation 16 of
+            :cite:t:`mewe_freebound_1986`.
+
+        Notes
+        -----
+        The input ion `self` is taken to the recombining ion.
+
+        The calculation does not include the abundances and ionization equilibria, unlike in :cite:t:`mewe_freebound_1986`.
+
+        Equation 14 of :cite:t:`mewe_freebound_1986` has a simple
+        analytic solution.  They approximate
+        .. math::
+            f_{1}(Z, n, n_{0} ) = \sum_{1}^{\infty} n^{-3} - \sum_{1}^{n_{0}} n^{-3} = \zeta(3) - \sum_{1}^{n_{0}} n^{-3} \approx 0.21 n_{0}^{-1.5}
+
+        where :math: `\zeta(x)` is the Riemann zeta function.
+
+        However, the second sum is analytic, :math: `\sum_{1}^{n_{0}} n^{-3} = \zeta(3) + \frac{1}{2}\psi^{(2)}(n_{0}+1)`
+        where :math: `\psi^{n}(x)` is the n-th derivative of the digamma function (a.k.a. the polygamma function).
+        So, we can write the full solution as:
+        .. math::
+            f_{1}(Z, n, n_{0}) = \zeta(3) - \sum_{1}^{n_{0}} n^{-3} = - \frac{1}{2}\psi^{(2)}(n_{0}+1)
+
+        The final expression is therefore simplified and more accurate than :cite:t:`mewe_freebound_1986`.
```

Comment:
Because this function is not public API (which I agree with!), this docstring does not show up in the rendered documentation. Should we maybe move this elsewhere in the docs? This seems like useful information to record somewhere. We could potentially leave this to a later PR and just leave it as is for now.

Reply:
I didn't realize this wouldn't show up.  This absolutely needs to be recorded since we've changed the implementation slightly from Chianti, which uses the polynomial form.  I'm not sure where else to suggest that it be moved to, though.

Reply:
The latest version of the docs has a section called "Topic Guides" that is reserved for more non-code explanatory pages: https://fiasco.readthedocs.io/en/latest/topic_guides/index.html. We could put it there.

Reply:
Alright, I've added it now.  Hopefully I've formatted it correctly . . . 

Also, thanks for catching the bug in the test!  Beat me to it.